tests/unit/mesh/data/test_gridfromoptions.cxx

```diff
@@ -62,6 +62,12 @@ class GridFromOptionsTest : public ::testing::Test {
           return index.x() + (TWOPI * index.y()) + (TWOPI * index.z() / nz) + 3;
         },
         &mesh_from_options);
+    expected_metric =
+#ifdef COORDINATES_USE_3D
+        expected_3d;
+#else
+        expected_2d;
+#endif
```

Comment:
When `bout::build::use_3d_metric` is added, can use that with a ternary to avoid splitting the statement with a macro